vit_jax/models_mixer.py

```diff
@@ -64,5 +64,7 @@ def __call__(self, inputs, *, train):
       x = MixerBlock(self.tokens_mlp_dim, self.channels_mlp_dim)(x)
     x = nn.LayerNorm(name='pre_head_layer_norm')(x)
     x = jnp.mean(x, axis=1)
-    return nn.Dense(self.num_classes, kernel_init=nn.initializers.zeros,
+    if self.num_classes:
+      x = nn.Dense(self.num_classes, kernel_init=nn.initializers.zeros,
```

Comment:
nit: Could you intend line 69 two more spaces accordingly?

Reply:
@andsteing followed the convention of https://github.com/sayakpaul/vision_transformer/blob/mixer-b32/vit_jax/models_mixer.py#L61

Reply:
Looks good now.